### DISCUSSION TITLE
Can we get Rugby and Rugby league supported?

### DISCUSSION THREAD

**Discussion by Wheresitat** [vasqued2/ha-teamtracker#32] [General]
Great to see that the AFL is now added to Teamtracker. Is Rugby Union and Rugby League on the horizon? These are international sports, with world cups etc. Would be great to see.

Thanks for all you do.

**Comment by vasqued2** [comment]
Looks like it already supports it, at least the post-game results.  Just use the [Custom API Configuration](https://github.com/vasqued2/ha-teamtracker/wiki/Custom-API-Configurations) option.  

```
sport_path: rugby
league_path: unique_6_digit_league_code

```
Go [here](https://www.espn.com/rugby/scoreboard/_/league/244293) and look at the league drop-down for the list of leagues I would expect ESPN to support.  I have no idea what any of them are or if they work but hopefully they mean something to you.

I just tested league 244293 and it returned the results of the NZL/AUS game on Sept 24.  If there are errors while in-game, lmk.

**Reply by Wheresitat** [reply]
Ohk thanks very much, will give it a try and let you know how it goes

**Reply by Wheresitat** [reply]
Can confirm it seems to work, thanks again.

**Comment by vasqued2** [comment]
There isn't a working example of Rugby in the Wiki yet.  If you have a config that works and can add it, I'm sure it will help someone else.

**Reply by Wheresitat** [reply]
For Rugby league - https://prnt.sc/zhMvddwi9TEX here's the settings that i used.
Sport path "rugby-league"
League path will need to be "3"
